lexers/troff.lua

```diff
@@ -0,0 +1,33 @@
+-- Copyright 2015-2017 David B. Lamkins <david@lamkins.net>. See LICENSE.
+-- man/roff LPeg lexer.
+
+local lexer = require('lexer')
+local token, word_match = l.token, l.word_match
```

Comment:
`l` is not defined here. I think you mean `lexer` for this and all other instances in the file.

Reply:
This is fixed, and another typo too (`M` was returned instead of `lex`)